### PR TITLE
FLUID-5673: Fix and test case for failure to properly dispose of tooltips whose target doesn't match an element in "items"

### PR DESCRIPTION
**Pull request by amb26**

### CODE REVIEW

**Comment by amb26** [comment]
@jobara - please verify. I did find that the update in FD was still a little laggy but the tooltip accumulation problem appears to be resolved

File: src/components/tooltip/js/Tooltip.js [hunk 1]
```diff
@@ -55,6 +55,14 @@ var fluid_2_0 = fluid_2_0 || {};
             };
         }
     };
+    
+    // Resolve FLUID-5673 by resolving the event target upwards to the nearest match for "items" - this will
+    // reproduce the natural effect operated by event bubbling in conjunction with the widget
+    fluid.tooltip.resolveTooltipTarget = function (that, event) {
+        var originalTarget = fluid.resolveEventTarget(event);
+        var tooltipTarget = $(originalTarget).closest(that.options.items);
```
Comment (jobara) [inline src/components/tooltip/js/Tooltip.js]: Rather passing in the `that` can we just pass in `items` directly?

**Comment by jobara** [comment]
@amb26 I'm getting test failures from the Paged Table Test Suite regarding Tooltip visibility.

**Comment by jobara** [comment]
merged @ fcb1c18a9f4da94d6564a521175e772f2180e8c5

**Pull request merged**